recipes/r-monocle3/meta.yaml

```diff
@@ -1,19 +1,21 @@
-{% set version = "1.0.0" %}
+{% set version = "1.3.1" %}
 {% set github = "https://github.com/cole-trapnell-lab/monocle3" %}
 
 package:
   name: r-monocle3
   version: "{{ version }}"
 
 source:
-  url: "{{ github }}/archive/{{ version }}.tar.gz"
-  sha256: f09521a22b3584d000675e7ed659f2f41e823667bd019b38c52f6ab2ea654a00
+  url: "{{ github }}/archive/v{{ version }}.tar.gz"
+  sha256: 30e86d5323ea22b302614813ecf102f26774b42710589671a1f1d51ef9ad183d
 
 build:
-  number: 5
+  number: 0
   rpaths:
     - lib/R/lib/
     - lib/
+  run_exports:
+    - {{ pin_subpackage('r-monocle3', max_pin="x") }}
```

Comment:
`- {{ pin_subpackage('r-monocle3', max_pin="x.x") }}` since the version is `major.minor.patch`

Reply:
Good catch, I misunderstood the documentation! Thanks!